spec/services/doc_auth/lexis_nexis/responses/true_id_response_spec.rb

```diff
@@ -315,6 +319,12 @@ def get_decision_product(resp)
         to match(a_hash_including(visible_pattern: { no_side: 'Failed' }))
     end
 
+    it 'returns Failed for livness failure' do
+      output = described_class.new(failure_response_with_liveness, config).to_h
+      Rails.logger.debug output.to_h.to_json
```

Comment:
What's the purpose of including the `Rails.logger.debug` statement?

Reply:
@eileen-nava  ah, that's my IDE's auto correction. will remove it.